### PR TITLE
Re-expose persistent subscription connect function.

### LINKED ISSUE #82: No method to connect to PersistentSubscriptionAll
Hi,

I can successfully create a persistent subscription that listens to specific events by doing: 
```
db.CreatePersistentSubscriptionAll(
	context.Background(),
	"GroupName",
	esdb.PersistentAllSubscriptionOptions{
		Filter: &esdb.SubscriptionFilter{
			Type:     esdb.EventFilterType,
			Prefixes: []string{"EventType-"},
		},
	},
)
```
But I could not find the method to subscribe to it without passing a stream name:
```
db.ConnectToPersistentSubscription(
   context.Background(),
   ???, 
   "GroupName", 
   esdb.ConnectToPersistentSubscriptionOptions{}
)
```
Am I missing something or it's just not implemented yet?

### PR DESCRIPTION
**Pull request by YoEight**
Fixes #82

### CODE REVIEW

**Comment by ylorph** [comment]
@YoEight : on windows all  tests pass  without issue for the matrix 20.10.4-buster-slim, 21.6.0-buster-slim, ci

**Review by ylorph — COMMENTED** [review]
ok for me , 
 ! we need to create an issue for the naming convention `connect` vs `subscribe`
 for both the persistent subcription to all & to a group/stream 
 
 that should'nt block the merge imho

**Review by ylorph — CHANGES_REQUESTED** [review]
We need a note in the readme for the EVENTSTORE_DOCKER_TAG_ENV if people want to run the test against a specific database version .

File: esdb/client.go [hunk 1]
```diff
@@ -434,6 +434,28 @@ func (client *Client) ConnectToPersistentSubscription(
 	)
 }
 
+func (client *Client) ConnectToPersistentSubscriptionToAll(
```
Comment (George-Payne) [inline esdb/client.go:437]: The RFC calls this method `SubscribeToPersistentSubscriptionToAll`

https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/022%20-%20gRPC%20Persistent%20Subscriptions.md#supported-operations

Comment (George-Payne) [reply esdb/client.go:437]: Oh, just saw Yves's comment, ignore me.

Comment (George-Payne) [reply esdb/client.go:437]: Yeah, also needs to be changed in `ConnectToPersistentSubscription`, so good to do a single alignment PR.

**Comment by YoEight** [comment]
@George-Payne @ylorph The issue was created there: https://github.com/EventStore/EventStore-Client-Go/issues/85

**Review by ylorph — APPROVED** [review]

**Review by hayley-jean — APPROVED** [review]
This looks good, the names can be aligned in a different PR since there's also things like `BatchSize` vs `BufferSize` to check as well.

**Pull request merged**